src/views/CreateType.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+  <main class="create-type" v-if="formState === 0" @keydown.enter="nextStep">
+    <ViewHeader title="How should the resource type be called?" :backLink="{ link: '/types' }"/>
+    <Input :value.sync="newResourceType.name" name="Name" placeholder="Cars" :autofocus="true"/>
+    <Button text="Continue" :onClick="nextStep"/>
+  </main>
+
+  <main v-else-if="formState === 1">
+    <ViewHeader
+      :title="`Is ${newResourceType.name} an abstract resource type?`"
+      :backLink="{ onClick: previousStep }"
+    />
+    <Toggle :value.sync="newResourceType.abstract" name="Abstract"/>
+    <Button text="Continue" :onClick="nextStep"/>
+  </main>
+
+  <main v-else-if="formState === 2">
+    <ViewHeader
+      :title="`Choose a parent type for ${newResourceType.name}`"
+      :backLink="{ onClick: previousStep }"
+    />
+    <ListSection :list="parentTypeList"/>
+  </main>
+
+  <main v-else-if="formState === 3">
+    <ViewHeader
+      :title="`Define attributes for ${newResourceType.name}`"
+      :backLink="{ onClick: previousStep }"
+    />
+    <ListSection :title="`Attributes of ${parentType.name}`" :list="parentTypeAttributeList"/>
+
+    <ListSection :title="`Attributes of ${newResourceType.name}`">
+      <Li v-for="attribute in attributeListLeft" :key="attribute.id" :options="attribute"/>
```

Comment:
Here again, the naming issue appears 💥 🦄 !
Everything is written out, `ListSection` actually renders a `ul`, but `li` is named `Li`.